### DISCUSSION TITLE
Add a way to copy usernames for quicker reply in public chats

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#21826] [Ideas] [answered]
To clarify the idea: a simple button (in the context menu?) to copy a nickname or make it appear directly in the chat (but in that case, you would have to take into account whether one has already written something in the textbox).

This a small addition that might help save time writing someone's nickname (especially when it's a bit long). Generally, if I reply to someone, to keep track of them I write their name in the message (that way it should also notify the recipient if I'm not mistaken).
In some cases, it becomes cumbersome as a thing because I would have to write down the whole nickname by hand, checking capitalizations and various numbers (or use alternatives, such as opening his profile from the website and copying the name).

**Comment by peppy** [comment]
Would also be nice to have a right click -> "Reply" option to autofill the message window with `@User:`?

**Comment by Joehuu** [comment, marked as answer]
Will be added in https://github.com/ppy/osu/pull/22262.
